sprintf.c

```diff
@@ -246,6 +246,16 @@ rb_str_format(int argc, const VALUE *argv, VALUE fmt)
         rb_raise(rb_eArgError, "flag after precision"); \
     }
 
+#define update_coderange(partial) do { \
```

Comment:
is there a benefit to using a macro over `static inline void update_coderange (int partial) {...}` ?

Reply:
Nevermind, I missed that it access plenty of local variables.